src/Sdk/RSWebApi/RunServiceHttpClient.cs

```diff
@@ -174,5 +184,11 @@ public class RunServiceHttpClient : RawHttpClientBase
                     throw new Exception($"Failed to renew job: {result.Error}");
             }
         }
+
+        protected override async Task<T> ReadJsonContentAsync<T>(HttpResponseMessage response, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This is a virtual method on `RawHttpClientBase`

https://github.com/actions/runner/blob/04b07b6675c56da40e532ab73eeb4287bf223e34/src/Sdk/WebApi/WebApi/RawHttpClientBase.cs#L235